### DISCUSSION TITLE
How to: Add tooltips to all images

### DISCUSSION THREAD

**Discussion by wilhelmer** [squidfunk/mkdocs-material#4154] [Show and tell]
As you probably know, the syntax for adding images in Markdown is like this:

```markdown
![Alt text](assets/images/my-image.png "My title")
```

The **alt text** is helpful, e.g., for screen readers and SEO purposes. 

The **title** is helpful because most desktop browsers display it as a "tooltip" when you hover over the image.

![Tooltip Demo](https://j.gifs.com/6WvnnR.gif)

However, adding and maintaining **both** the alt text and the title on all of your images can be cumbersome.

This is why I (and probably most users?) usually omit the title and only specify the alt text:

```markdown
![Image description](assets/images/my-image.png)
```

The downside of this is that you're adding information that, while being super beneficial for a small, but important (!) group of users, most users won't ever notice. There's no tooltip.

Wouldn't it be great to display this information to all users, empowered by MkDocs Material's [improved tooltips](https://squidfunk.github.io/mkdocs-material/reference/tooltips/#improved-tooltips)?

Here's how:

1. Enable [content.tooltips](https://squidfunk.github.io/mkdocs-material/reference/tooltips/#improved-tooltips).
2. Install [markdown-image-titles](https://github.com/wilhelmer/markdown-image-titles).

Done! That was easy, huh? 

Have a nice day!

**Comment by ryran** [comment]
Very cool! This should be default.
